plugins/modules/route53_health_check.py

```diff
@@ -45,8 +45,22 @@
       - The type of health check that you want to create, which indicates how
         Amazon Route 53 determines whether an endpoint is healthy.
       - Once health_check is created, type can not be changed.
-    choices: [ 'HTTP', 'HTTPS', 'HTTP_STR_MATCH', 'HTTPS_STR_MATCH', 'TCP' ]
+    choices: [ 'HTTP', 'HTTPS', 'HTTP_STR_MATCH', 'HTTPS_STR_MATCH', 'TCP', 'CALCULATED' ]
     type: str
+    version_added: 6.3.0
```

Comment:
The `version_added` here suggests that the `type` parameter was added in 6.3.0, which is not true. I think what @alinabuzachis was suggesting is that instead of this, add a line in the description that says the `CALCULATED` choice was added in 6.3.0.